include/networkit/algebraic/CSRGeneralMatrix.hpp

```diff
@@ -56,11 +56,30 @@ class CSRGeneralMatrix {
     index binarySearchColumns(index left, index right, index j) const {
```

Comment:
The code and documentation here do not match. Please check if  we have any code in our project that expects / deals with `binarySearchColumns` returning `none`?


Also, there is an undefined special case in the documentation: what is the return value if we search in the first row for a column that does not exist and there is no "index immediately left of the place where j would be"? (this would be index `-1`?). Currently our code returns `none`, but that is not documented.

Reply:
This function is only used in the 'CSRGeneralMatrix.hpp', the doc is now adjusted